TabloidCLI/UserInterfaceManagers/JournalManager.cs

```diff
@@ -149,6 +149,9 @@ private void Remove()
             {
                 _journalRepository.Delete(entryToDelete.Id);
             }
+
+            Console.WriteLine($"Your entry '{entryToDelete.Title}' has been removed.");
```

Comment:
You should move this line inside the if block incase the user changes mind and doesn't enter an entry to delete. Otherwise, it will throw an exception because the reference to entryToDelete will be null.